assets/aml-benchmark/components/src/dataset_downloader/loaders/math.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
Also, Eric mentioned about CELA requirements for adding a notice.txt file. Please make sure those requirements are properly implemented.

Reply:
@EricWrightAtWork, could you please help in understanding what needs to be included to meet the requirements? Thanks!